.github/workflows/vapoursynth.yml

```diff
@@ -0,0 +1,151 @@
+name: vapoursynth
+
+on: [push, pull_request]
+
+jobs:
+  clippy-rustfmt:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+        components: clippy, rustfmt
+    - name: Run clippy
+      uses: actions-rs/clippy-check@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        args: --all --all-features
```

Comment:
So since
- `sample-plugin` can't be built with `--all-features` (semi-deliberately; it's made to support the minimal API version, and I never added the `#[cfg()]` there for higher API versions), and
- we want to check the examples and tests too,

this should be replaced with two consecutive clippy invocations:
- `cargo clippy --all --all-targets --all-features --exclude sample-plugin`
- `cargo clippy --all-targets --package sample-plugin`

Reply:
Ok, implemented the two `cargo clippy` steps. Thanks!